packages/google_mlkit_commons/android/src/main/java/com/google_mlkit_commons/InputImageConverter.java

```diff
@@ -53,9 +61,36 @@ public static InputImage getInputImageFromData(Map<String, Object> imageData,
                                 rotationDegrees,
                                 imageFormat);
                     }
+                    if (imageFormat == ImageFormat.YUV_420_888) {
+                        // This image format is only supported in InputImage.fromMediaImage, which requires to transform the data to the right java type.
+                        // TODO: Consider reusing the same Surface across multiple calls to save on allocations.
```

Comment:
Reusing this ImageWriter across multiple calls, thus avoiding the allocation, would make this likely more performant. Could any of the reviewers take a look? 

Is it something that could be left as a TODO and fixed later?